src/test/java/harness/TestConfig.java

```diff
@@ -24,6 +24,12 @@ public final class TestConfig {
   // see PF-886.
   private boolean useJanitorForExternalResourcesCreatedByTests;
 
+  // The PubSub topic to use for publishing Janitor cleanup requests
+  private String janitorPubSubTopic;
```

Comment:
Optional - This is the same for all envs right? Can this be a constant instead of test config?

Reply:
It's the same for all Broad environments but would be different for a Verily deployment, so I think this needs to live in TestConfig